resources/js/Pages/Dashboard/CourseSemester/Index.vue

```diff
@@ -0,0 +1,92 @@
+<script setup>
+import DashboardLayout from '@/Layouts/DashboardLayout.vue'
+import TableHeader from '@/Shared/Components/Table/TableHeader.vue'
+import TableWrapper from '@/Shared/Components/Table/TableWrapper.vue'
+import TableRow from '@/Shared/Components/Table/TableRow.vue'
+import TableCell from '@/Shared/Components/Table/TableCell.vue'
+import Button from '@/Shared/Components/Buttons/Button.vue'
+import EmptyState from '@/Shared/Components/EmptyState.vue'
+import RemoveModal from '@/Shared/Modals/RemoveModal.vue'
+import { ref } from 'vue'
+import ManagementHeader from '@/Shared/Components/ManagementHeader.vue'
+import ManagementHeaderItem from '@/Shared/Components/ManagementHeaderItem.vue'
+import { Cog6ToothIcon, XCircleIcon, EyeIcon } from '@heroicons/vue/24/outline'
+
+defineProps({
+  courses: Object,
+  total: Number,
+  lastUpdate: String,
+})
+const showModal = ref(false)
+const courseToDeleteId = ref(0)
+</script>
+
+<template>
+  <DashboardLayout>
+    <div class="flex flex-col gap-8">
+      <ManagementHeader>
+        <template #header>
+          Zarządzanie kursami w semestrze
+        </template>
+        <template #statistics>
+          <ManagementHeaderItem>
+            Liczba kursów w bazie: {{ total }}
+          </ManagementHeaderItem>
+          <ManagementHeaderItem v-if="lastUpdate">
+            Ostatnie zmiany: {{ lastUpdate }}
+          </ManagementHeaderItem>
+        </template>
+        <template #actions>
+          <span class="hidden sm:block">
+            <Button :href="`/dashboard/course-semester/create`">
+              Dodaj
+            </Button>
+          </span>
+        </template>
+      </ManagementHeader>
+      <div v-if="courses.data.length" class="flex flex-col gap-8">
+        <TableWrapper>
+          <template #header>
+            <TableHeader class="w-1/6">
+              ID
+            </TableHeader>
+            <TableHeader class="w-1/6">
+              Kurs
+            </TableHeader>
+            <TableHeader class="w-1/6">
+              Semestr
+            </TableHeader>
+            <TableHeader />
```

Comment:
I would add here number of assigned group indicator.

Reply:
added